cli/src/main/java/com/devonfw/tools/ide/tool/GlobalToolCommandlet.java

```diff
@@ -143,4 +143,10 @@ protected boolean doInstall(boolean silent) {
     postInstall();
     return true;
   }
+
+  @Override
+  public void uninstall() {
+    //TODO: handle "uninstall <globaltool>"
```

Comment:
For the moment I also do not see a better quickfix.
However, could you be so nice and create a new issue about implementing uninstall for global tools and link it in this comment?

Reply:
And easy alternative would be that we never support uninstalling global tools automatically and change the `UninstallCommandlet` to only accept `LocalToolCommandlet`s by crafting an according property for that.

Reply:
I think it would be best to keep UninstallCommandlet to accept GlobalToolCommandlets for the following two reasons:
For cases where uninstalling global tools by IDEasy is not be supported, this method can be utilized to provide reasonable feedback to the user (i.e. instructions on how to manually uninstall), before stating the error "Couldn't uninstall [..]".  
For global tools installed by package managers (see e.g. docker for Linux), uninstalling global tools can be supported by IDEasy.

Reply:
> For the moment I also do not see a better quickfix. However, could you be so nice and create a new issue about implementing uninstall for global tools and link it in this comment?

Created #399

Reply:
> I think it would be best to keep UninstallCommandlet to accept GlobalToolCommandlets for the following two reasons:

OK

> For cases where uninstalling global tools by IDEasy is not be supported, this method can be utilized to provide reasonable feedback to the user (i.e. instructions on how to manually uninstall), before stating the error "Couldn't uninstall [..]".

The user could then not assign "docker" or "pgadmin" as argument to the `uninstall` commandlet and tab-completion would not suggest global tools. If the user manually types these as arguments the CLI would raise something like "invalid argument 'docker' for local tool".
Still I agree that an average user may not even know the difference between local and global tools.

> For global tools installed by package managers (see e.g. docker for Linux), uninstalling global tools can be supported by IDEasy.

On windows this may be quite difficult to automate and MacOS is missing the concept to uninstall an application. Apple just forgot about this idea and still argue that you can simply go to your `Application` folder and manually delete the app. What they forget to see is that such Application can do several tweaks to your computer (e.g. writing changes to `.bashrc`, registering workflows, etc.) that are not undone when you simply delete the app folder.

However, let us just agree that we also want to support uninstall for global tools in #399 as good as we can do.
I therefore resolve this review comment and merge this PR. Thanks @slskiba for your great work. 👍